web/src/scenes/app/Configuration.tsx

```diff
@@ -29,37 +29,37 @@ const saveSchema = yup.object().shape({
     .string()
     .max(3500)
     .required("This section is required"),
-  description_how_it_works: yup.string().max(3500).notRequired(),
-  description_connect: yup.string().max(3500).notRequired(),
-  link_to_integration: yup
+  description_how_it_works: yup.string().max(3500).optional(),
+  description_connect: yup.string().max(3500).optional(),
+  integration_url: yup
     .string()
-    .notRequired()
     .url("Must be a valid URL")
-    .matches(/^https:\/\/|^$/, "Link must start with https://"),
+    .matches(/^https:\/\/|^$/, "Link must start with https://")
+    .optional(),
   world_app_description: yup
     .string()
     .max(50, "In app description cannot exceed 50 characters")
-    .notRequired(),
-  category: yup.string().default("").notRequired(),
+    .optional(),
+  category: yup.string().default("").optional(),
   is_developer_allow_listing: yup.boolean().default(false),
 });
 
 type ConfigurationFormValues = yup.Asserts<typeof saveSchema>;
 
 export const Configuration = memo(function Configuration() {
   const currentApp = useAppStore((store) => store.currentApp);
-  const { updateAppData, parseDescription, encodeDescription } = useApps();
+  const { updateAppMetadata, parseDescription, encodeDescription } = useApps();
 
-  const descriptionInternal = parseDescription(currentApp);
+  const descriptionInternal = parseDescription(currentApp?.app_metadata);
```

Comment:
this is a different description, let's change to make sure we're updating the public description only right now